src/ProjectOrigin.WalletSystem.Server/ProjectOrigin.WalletSystem.Server.csproj

```diff
@@ -7,19 +7,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="Database\Scripts\202305082048-create-tables.sql" />
+    <EmbeddedResource Include="Database\Scripts\202305101334-create-wallet-tables.sql" />
   </ItemGroup>
 
   <ItemGroup>
-    <Protobuf Include="..\Protos\primitives.proto" Access="internal"/>
-    <Protobuf Include="..\Protos\wallet.proto" GrpcServices="Server" Access="internal"/>
+    <Protobuf Include="..\Protos\primitives.proto" />
+    <Protobuf Include="..\Protos\wallet.proto" GrpcServices="Both" />
```

Comment:
Why is this set to "Both"?

Reply:
Kind of, I changed it so both the server and client classes are generated in the WalletSystem.Server, otherwise we would need to have this in both csproj files, AND they are **required to be internal** as not to make namespaces clashes, which in turn requires a lot of the classes in the WalletSysten.Server to be internal instead of public, which creates an issue when we want to test classes from the other project, and **if we then enable internal classes to be visible for the test project, then we have conflicts agin.**